woocommerce/wc-admin
# Add 'isCheckbox' prop to 'MenuItem'

Fixes #1609.

- Adds a new prop `isCheckbox` to `<MenuItem>` that allows creating menu items with a `FormToggle` and [`role=menuitemcheckbox`](https://www.w3.org/TR/wai-aria-1.1/#menuitemcheckbox).
- Updates all menus that were using `<MenuItem>` with [`role=menuitem`](https://www.w3.org/TR/wai-aria-1.1/#menuitem) and had a checkbox inside that, to use the new prop.

### Accessibility


### Screenshots
![image](https://user-images.githubusercontent.com/3616980/53014251-3b60cc80-3448-11e9-99f0-11133e5a161c.png)

### Detailed test instructions:
* Go to any report.
* Tab to columns menu (vertical ellipse).
* Verify only menu items are focusable but toggle buttons are not (before, you needed two Tabs to jump from one option to the next).
* Toggle a column off with the keyboard.
* Verify the column can be toggled back on with the keyboard.
* Repeat the same process with the _Dashboard_ menus: _Store Performance_, _Leaderboards_ and _Charts_.
